build.gradle

```diff
@@ -33,7 +33,7 @@ subprojects {
     }
 
     ext {
-        slf4jVersion = "1.7.32"
+        slf4jVersion = "2.0.0"
```

Comment:
Seems `2.0.1` has been just published

Reply:
Actually, I noticed it's not a great idea to use slf4j 2+ due to https://github.com/spring-projects/spring-boot/issues/12649 (current spring-boot versions cannot support it). Let me submit another PR to downgrade it to latest 1.X